django_gcp/tasks/tasks.py

```diff
@@ -83,13 +83,23 @@ class Task(metaclass=TaskMeta):
     deduplicate = False
 
     def enqueue(self, **kwargs):
-        """Invoke a task (place it onto a queue for processing)"""
+        """Enqueues the given task for processing, unless 'self.manager.disable_execution' is True.
+        If `manager.disable_execution` is True, this method returns None without enqueuing the task.
+        """
+        if self.manager.disable_execution:
+            return None
+
```

Comment:
This is duplicated code. If you look at the way the `eager_execute` option is exercised, you'll see that a common place for this return statement might be in the `_send` method.

Reply:
Moved code into _send method